src/main/java/org/spongepowered/api/statistic/achievement/Achievement.java

```diff
@@ -75,89 +60,17 @@ static Builder builder() {
     Collection<Achievement> getChildren();
 
     /**
-     * Gets the {@link Statistic} which this achievement is backed by if it is available.
+     * Returns the {@link ItemStack} that this achievement is represented by.
      *
-     * @return The source statistic, if available
+     * @return ItemStack representation
      */
-    Optional<Statistic> getSourceStatistic();
+    Optional<ItemStack> getItemStack();
```

Comment:
Likewise, yes, I'd recommend using `ItemStackSnapshot` to avoid having to constantly copy the `ItemStack` defensively when returning it.